ctx.go

```diff
@@ -37,6 +52,18 @@ type ContextWithBody[B any] interface {
 	// MustBody works like Body, but panics if there is an error.
 	MustBody() B
 
+	// Params returns the typed parameters of the request.
+	// It returns an error if the parameters are not valid.
+	// Please do not use a pointer type as parameters.
+	//
+	// Deprecated: Not defined yet, incoming in future Fuego versions.
+	Params() (P, error)
+
+	// MustParams works like Params, but panics if there is an error.
+	//
+	// Deprecated: Not defined yet, incoming in future Fuego versions.
+	MustParams() P
```

Comment:
Why adding them?

Reply:
To split this big PR into several readable PRs